spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/reload/ConfigReloadUtil.java

```diff
@@ -110,6 +111,10 @@ else if (source instanceof MountConfigMapPropertySource mountConfigMapPropertySo
 				// we know that the type is correct here
 				managedSources.add((S) mountConfigMapPropertySource);
 			}
+			else if (source instanceof SecretsPropertySource secretsPropertySource) {
```

Comment:
when we were mounting secrets, reload was not working, since there was no this check in place. Even it I'm fixing it here, it will eventually be dropped, since we will move from mounting to config import.